tappsbt/interface.go

```diff
@@ -50,6 +50,7 @@ var (
 	PsbtKeyTypeOutputTapAsset                              = []byte{0x76}
 	PsbtKeyTypeOutputTapSplitAsset                         = []byte{0x77}
 	PsbtKeyTypeOutputTapAnchorTapscriptSibling             = []byte{0x78}
+	PsbtKeyTypeOutputAssetVersion                          = []byte{0x79}
```

Comment:
Needs an update to the BIP.

Reply:
https://github.com/Roasbeef/bips/pull/42